.travis.yml

```diff
@@ -1,12 +1,19 @@
 language: node_js
+sudo: false
 node_js:
   - "6"
   - "8"
   - "10"
-sudo: false
+  - "11"

```

Comment:
Perhaps using `lts/*` and/or `stable` would be better than manually specifying the version (which could be subject to change in the future) would be a better idea no?

Reply:
really not sure about this.

Reply:
Have a look at [this repo](https://travis-ci.org/Berkmann18/TemplateJS/) which shows how you could get your code to be tested on the latest LTS version and stable version of NodeJS regardless of the year.

Reply:
I know it's possibile, what I mean if I doubt this is a good thing to do

Reply:
Fair enough.